make/autoconf/platform.m4

```diff
@@ -226,6 +226,10 @@ AC_DEFUN([PLATFORM_EXTRACT_VARS_FROM_OS],
       VAR_OS=aix
       VAR_OS_TYPE=unix
       ;;
+    *ztpf*)
+      VAR_OS=ztpf
+      VAR_OS_TYPE=unix
+      ;;
```

Comment:
Unfortunately, this would only represent the tip of this particular iceberg. I expect many more changes would be required for proper configuration of builds for z/TPF. Please revert this change.

Reply:
Update to platform.m4 reverted.